### PR TITLE
Add a shortcut to the GitHub integration page from the repos page.

### PR DESCRIPTION
**Pull request by frewsxcv**
Fixes https://github.com/bors-ng/bors-ng/issues/93.

### CODE REVIEW

File: apps/bors_frontend/web/templates/project/index.html.eex [hunk 1]
```diff
@@ -16,6 +16,12 @@
 </div>
 <% end %>
 
+<div class=toolbar>
+  <a class=toolbar-item href="https://github.com/integrations/bors-ng/installations/new">
```
Comment (frewsxcv) [inline apps/bors_frontend/web/templates/project/index.html.eex]: Maybe this URL should be specified in a config?

Comment (notriddle) [inline apps/bors_frontend/web/templates/project/index.html.eex]: Yes, it probably should.

**Comment by frewsxcv** [comment]
Finally got around to this. Let me know how it can be improved

**Pull request merged**